gestalt-asset-core/src/main/java/org/terasology/gestalt/assets/format/AbstractAssetAlterationFileFormat.java

```diff
@@ -34,17 +34,26 @@ public abstract class AbstractAssetAlterationFileFormat<T extends AssetData> imp
     private Predicate<FileReference> fileMatcher;
 
     /**
+     * Creates a AssetAlterationFileFormat that will handle files with given extensions.
      * @param fileExtension  A file extension that this file format will handle
      * @param fileExtensions Additional file extensions that this file format will handle
      */
     public AbstractAssetAlterationFileFormat(String fileExtension, String... fileExtensions) {
         this.fileMatcher = FileUtil.createFileExtensionPredicate(Varargs.combineToList(fileExtension, fileExtensions));
     }
 
+    /**
+     * Creates a AssetAlterationFileFormat that will handle files with specified file extension.
```

Comment:
```suggestion
     * Creates a AssetAlterationFileFormat that will handle files matching the given predicate.
```
This description is almost right. We do not provide a file extension for this overload though, instead using a predicate for filtering instead.